### DISCUSSION TITLE
graphql AND condition

### DISCUSSION THREAD

**Discussion by bfourgeaud** [strapi/strapi#6408] [Legacy Topics]
Hello,

I saw on the documentation that strapi doesn't allow AND condition on the graphql plugin.
I was wondering how I could do to find objects in my database that matches both conditions.

I have :

```
travels:[
 {
  name: "France 1",
  cities: [ {name:"Lyon"},  {name:"Paris"} ]
 },
 {
  name: "France 2",
  cities: [ {name:"Lyon"},  {name:"Marseille"} ]
 }
]
```

How could I find all the travels which are going through both cities Paris & Lyon ?

I tried whithout success : 
```
"cities": {
      "name_in": "Paris",
      "name_in": "Lyon"
    }
```

I was unable to find any way of using a filter like javascript would do with **Array.every()**

Is that a known issue, or is there a solution to that ?
Thanks,

**Comment by derrickmehaffy** [comment]
Hello,

I see you are wanting to ask a question that is not really a bug report or a feature request, questions should be directed to the [discussions](https://github.com/strapi/strapi/discussions)

I have converted this to a discussion for you, we try to keep Github clean and only for bug reports and feature requests.

**Reply by bfourgeaud** [reply]
Sorry my fault ...
Thank you for converting that thread !

**Comment by corvallo** [comment]
Hi,
I'm facing a similar issue, but with or logical operator. Did you managed to solve this?

**Reply by bfourgeaud** [reply]
Hello, I didn't manage to solve this. I make the filtering client side for now, waiting for and update on my question ...
But from what I understood, the default behaviour id the OR logical operator, so you shouldn't have any problems.
